libopenage/renderer/camera/camera.cpp

```diff
@@ -113,6 +80,15 @@ void Camera::move_rel(Eigen::Vector3f direction, float delta) {
 	this->move_to(this->scene_pos + (direction * delta));
 }
 
+void Camera::move_rel(Eigen::Vector3f direction, float delta, CameraBoundaries camera_boundaries) {
+	auto new_pos = calc_look_at(this->scene_pos + (direction * delta));
```

Comment:
`move_rel` should not use `calc_look_at` because `move_rel` just moves the camera relative to it's previous position. Calling this would make the camera look at it's own position, which is unnecessary.